storage/innobase/xtrabackup/utils/debian/percona-xtrabackup-24.postinst

```diff
@@ -0,0 +1,74 @@
+#!/bin/sh
+
+set -e
+case "$1" in
+  configure)
+    BIN_PATH=/opt/percona-xtrabackup/2.4/usr/bin
+    DOC_PATH=/opt/percona-xtrabackup/2.4/usr/share/man/man1
+    cmd=''
+    man_cmd=''
+    FILES=$BIN_PATH/*
+    for f in $FILES
+    do
+      file=/usr/bin/$(basename $f)-24
+      if [ -L $file ];
+      then
+        rm -rf $file
+      fi
+      ln -s $f $file
+    done
+    FILES=$DOC_PATH/*
+    for f in $FILES
+    do
+      FileName=$(basename $f)
+      file=/usr/share/man/man1/$FileName-24
+      if [ -L $file ];
+      then
+        rm -rf $file
+      fi
+      ln -s $f $file
+      man_cmd=" $man_cmd --slave /usr/share/man/man1/$FileName $FileName $file"
+    done
+    for f in /usr/bin/xtrabackup /usr/bin/innobackupex /usr/bin/xbstream /usr/bin/xbcrypt /usr/bin/xbcloud /usr/bin/xbcloud_osenv
```

Comment:
for f in /usr/bin/{xtrabackup,innobackupex,...,...}?

Reply:
it works for /bin/bash but not for /bin/sh :(